CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing to Azure DevOps Extension
+
+Welcome, and thank you for your interest in contributing to Azure DevOps Extension!
+
+There are many ways in which you can contribute, beyond writing code. The goal of this document is to provide a high-level overview of how you can get involved.
+
+# Have a question
+
+Search existing github issues for similar questions and feel free to file an issue with a "question" tag to get our attention. 
+
+# Found a Bug
+
+#### Where to Find Known Issues
+
+Before you file a new bug report do check on existing open [issues](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
+
+If you find your issue already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment.
+👍 - upvote 👎 - downvote
+
+#### Reporting New Issue
+
+The best way to get your bug fixed is to provide good repro steps. Use our [Bug template](https://github.com/Microsoft/vsts-cli/issues/new/choose) to report any issues.
+
+#### Security Bugs
```

Comment:
Remove all TO-DO sections if you are targeting the main branch to be available on Day 0. If this is WIP, then its fine.